pkg/annotation/organization.go

```diff
@@ -0,0 +1,9 @@
+package annotation
+
+// support:
```

Comment:
I know this is probably copied but this godoc makes me sad.

Reply:
Just copied blindly :smile: Good point, I'll clean these up a bit.